coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -90,32 +147,36 @@ def form_valid(
             end_date_extension=10,
         )
 
-        # NOTE - "storage_protocols" will have special handling
-        attributes_to_check = [
```

Comment:
Are we only changing storage quotas and protocols now?  I haven't qa tested yet, but does this still allow for changing contacts, cost centers, etc.?